cocos/platform/win32/CCFileUtils-win32.cpp

```diff
@@ -168,8 +168,12 @@ static Data getData(const std::string& filename, bool forString)
 
         if (!successed)
         {
-            free(buffer);
-            buffer = nullptr;
+            // should determine buffer value, or it will cause memory leak
```

Comment:
@joewan 
You are right. If the pointer is null, it will do nothing.
But change it like this make it more elegant. 

@Japarazzi 
Sorry, i misunderstood it.
I will fix it now.